File: problems/easy/discount/discount.c
#include <stdio.h>
#include <math.h>

float cal_discount(float price, float percentage){
    // put your code here
    float discount = price * (percentage / 100)
    float finalPrice = price - discount;

    return finalprice;
}

int main(){
    float price, discount, finalPrice;
    
    printf("Enter original price: ");
    scanf("%d", &price);
    
    printf("Enter percentage discount: ");
    scanf("%f", &percentage);

    finalPrice = cal_dicsount(price, percentage); 
    printf("Final price: ");
    scanf("%d", finalPrice);
    
 return 0;
}

File: problems/easy/discount/discount.java
package problems.easy.discount;

public class discount {
    public static double get_discount(float p,float d){

        // write you code here
        double finalPrice = p - (p * d/100);
        return finalPrice;
    }
}
public static void main(String[] args){
    System.out.println(get_discount(1500,50));
}
